src/lib/Timeline.js

```diff
@@ -498,7 +500,10 @@ export default class ReactCalendarTimeline extends Component {
   }
 
   componentWillReceiveProps (nextProps) {
-    const { visibleTimeStart, visibleTimeEnd, items, groups, sidebarWidth } = nextProps
+    const { visibleTimeStart, visibleTimeEnd, items, groups, sidebarWidth, scrollableContainer } = nextProps
+    if (scrollableContainer !== this.props.scrollableContainer) {
+      throw new Error('scrollableContainer update is not implemented')
```

Comment:
What do you think about setting `this.scrollableContainer` in the constructor from props and never update `props.scrollableContainer`? We can be explicit in the README that this prop is a onetime setting?

I just don't like the idea of throwing and borking a user's app if they change a prop. I do think we should have a `console.warn` to tell the user that they're doing something wrong.